src/hotspot/share/runtime/os.hpp

```diff
@@ -296,6 +296,10 @@ class os: AllStatic {
   // Return the default page size.
   static int    vm_page_size();
 
+  // The largest possible standard page size (sysconf(_SC_PAGESIZE)) across all our
+  // platforms, usable as a compile-time constant.
+  static const size_t max_page_size_crossplatform = 64 * K; // ppc has 64 K pages.
+
```

Comment:
What is this for??

Reply:
A remnant of an older implementation that slipped through. Thanks for catching, I'll remove it.